ladder/teams/README.md

```diff
@@ -0,0 +1,51 @@
+# How Cilium Org Teams Are Managed
+Each Cilium org team that wishes to self-manage its team membership can do so using this directory.
+
+Each team can establish a file in this directory managed by the CodeOwners system with the assistance of 
+the Org admins (or duly deputized members of the org tasked with helping initiating team self-management). 
+
+On merge into the main branch (after appropriate approvals and review) GitHub automation 
+will propogate updates from the team files into the Cilium Organization's team membership using the GitHub API.
+
+## Are You a Team Member or a Mentor? Choose, but Choose Wisely.
```

Comment:
I'm guessing everyone would start as a team member then maybe move to mentor so I'm not sure this is really a choice at the beginning?